contrib/DEVELOPMENT_WORKFLOW.md

```diff
@@ -0,0 +1,165 @@
+# Bittensor Subnet Development Workflow
+
+This is a highly advisable workflow to follow to keep your subtensor project organized and foster ease of contribution.
+
+## Table of contents
+
+- [Bittensor Subnet Development Workflow](#bittensor-subnet-development-workflow)
+  - [Main Branches](#main-branches)
+  - [Development Model](#development-model)
+      - [Feature Branches](#feature-branches)
+      - [Release Branches](#release-branches)
+      - [Hotfix Branches](#hotfix-branches)
+  - [Git Operations](#git-operations)
+      - [Creating a Feature Branch](#creating-a-feature-branch)
+      - [Merging Feature Branch into Staging](#merging-feature-branch-into-staging)
+      - [Creating a Release Branch](#creating-a-release-branch)
+      - [Finishing a Release Branch](#finishing-a-release-branch)
+      - [Creating a Hotfix Branch](#creating-a-hotfix-branch)
+      - [Finishing a Hotfix Branch](#finishing-a-hotfix-branch)
+  - [Continuous Integration (CI) and Continuous Deployment (CD)](#continuous-integration-ci-and-continuous-deployment-cd)
+  - [Versioning and Release Notes](#versioning-and-release-notes)
+  - [Pending Tasks](#pending-tasks)
+
+## Main Branches
+
+Bittensor's codebase consists of two main branches: **master** and **staging**.
```

Comment:
Bittensor's codebase consists of two main branches: **main** and **staging**.

Reply:
Ahh, thanks!